### DISCUSSION TITLE
PrimeNG multiselect sort items on selection

### DISCUSSION THREAD

**Discussion by Sajeel-Bongale** [primefaces/community#346] [PrimeNG] [answered]
Is it possible to sort elements after selection in a multiselect such that they always appear alphabetically sorted?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Unfortunately, multiselect component does not offer such a feature.

**Reply by Sajeel-Bongale** [reply]
Thanks for your quick response!
